aggregator/src/main/java/com/redhat/cloud/notifications/db/EmailAggregationRepository.java

```diff
@@ -34,23 +39,27 @@ public List<AggregationCommand> getApplicationsWithPendingAggregationAccordinfOr
         List<Object[]> records = hqlQuery.getResultList();
         return records.stream()
                 .map(emailAggregationRecord -> new AggregationCommand(
-                    new EmailAggregationKey((String) emailAggregationRecord[0], (String) emailAggregationRecord[1], (String) emailAggregationRecord[2]),
-                    (LocalDateTime) emailAggregationRecord[3],
+                    new EmailAggregationKey((String) emailAggregationRecord[0], (String) emailAggregationRecord[1], (String) emailAggregationRecord[2], (String) emailAggregationRecord[2]),
```

Comment:
I guess it's a copy/paste typo :D 

```suggestion
                    new EmailAggregationKey((String) emailAggregationRecord[0], (String) emailAggregationRecord[1], (String) emailAggregationRecord[2], (String) emailAggregationRecord[3]),
```

Reply:
Yes it is, I fixed it on local, I will wait to go further on my end to end validation tests before pushing an update :)

Reply:
I just saw it and I know how frustrating can be these kind of typos so i had to tell :-)